crates/gpui/Cargo.toml

```diff
@@ -72,6 +72,9 @@ usvg = { version = "0.14", features = [] }
 util.workspace = true
 uuid = { version = "1.1.2", features = ["v4"] }
 waker-fn = "1.1.0"
+gtk-sys = "0.18.0"
```

Comment:
Seems like we should only depend on these crates on Linux?

Reply:
Right. Fixed it. Thanks!